packages/@sfx/sayt-driver-plugin/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@sfx/sayt-driver-plugin",
+  "version": "0.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "module": "src/index.ts",
+  "types": "dist/index.d.ts",
```

Comment:
Make sure to include `esnext`. Check the other packages for the correct order to keep it consistent.